### PR TITLE
Cellular and Data storage

### PR DESCRIPTION
**Pull request by emilybthorpe**
#11 #12

### CODE REVIEW

**Comment by emilybthorpe** [comment]
I'm going to mark this as ready for review for now (though I'm still waiting on the water sensor to be merged to integrate that into data collection) but the rest of the cellular and data systems are good to look at

**Comment by emilybthorpe** [comment]
But we should wait on merge until we have the watersensor merged

File: project.properties
Comment (ntlhui) [inline project.properties:3]: Are we still relying on `SpiffsParticleRK`?

File: src/cellular/dataCollection.cpp
Comment (ntlhui) [inline src/cellular/dataCollection.cpp:95]: Why are we accumulating after endianness change?  You must not do native math after endianness change, since the math is no longer valid.

File: src/cellular/dataCollection.cpp
Comment (ntlhui) [inline src/cellular/dataCollection.cpp:114]: Trailing space prior to semicolon

File: src/cellular/dataCollection.cpp
Comment (ntlhui) [inline src/cellular/dataCollection.cpp:101]: Why is this not populated?

File: src/cellular/dataCollection.cpp
Comment (ntlhui) [inline src/cellular/dataCollection.cpp:103]: Magic value needs to be in a #define

File: src/cellular/dataCollection.cpp
Comment (ntlhui) [inline src/cellular/dataCollection.cpp:114]: If there is accumulation happening, shouldn't there then be division at some point to get mean values? And shouldn't this only happen once everything is completely accumulated?

File: src/cellular/dataCollection.cpp
Comment (ntlhui) [inline src/cellular/dataCollection.cpp:98]: Why is this not getting encoded into the ensemble?

File: src/cellular/dataCollection.cpp
Comment (ntlhui) [inline src/cellular/dataCollection.cpp:1]: This does not behave the way https://github.com/UCSD-E4E/smartfin-fw2/blob/324b1749315e2d02d8b2e58c57178e95019c6c31/src/ride.cpp#L308 does.  Please implement the behavior from smartfin-fw2.

**Review by ntlhui — CHANGES_REQUESTED** [review]

**Comment by emilybthorpe** [comment]
Note - ride logic isnt in scope

**Pull request merged**
